docs/io.cozy.files.md

```diff
@@ -168,6 +168,63 @@ The `io.cozy.files` doctype has [the standard `cozyMetadata`](https://docs.cozy.
 }
 ```
 
+### Shortcuts
+
+Shortcuts (or hyperlinks) are a special type of files. They are served
+with the `application/shortcut` mime-type, they have a `.url` extension, and
```

Comment:
Je suis étonné du type mime. Le plus courant m'a l'air application/internet-shortcut. J'en trouve d'autres, mais aucun application/shortcut

Reply:
C'est ce que j'avais noté quand on en discuté ensemble : https://trello.com/c/bPmXyElo/371-%F0%9F%93%81-drive-%E2%98%81%EF%B8%8F-stack-%F0%9F%94%97-hyperlien-jh-2-3j#comment-5e330f048191660578b302d1, mais ça me va d'utiliser `application/internet-shortcut` (possible que je me sois trompé en prennant les notes).

Reply:
Je pense qu'on a du avoir une perte de communication quelque part. Pas très grave, la PR permet  de corriger